build-system/tasks/storybook/index.js

```diff
@@ -16,18 +16,19 @@ const {yellow} = require('kleur/colors');
 const ENV_PORTS = {
   amp: 9001,
   preact: 9002,
+  react: 9003,
```

Comment:
I vaguely remember we discussing this in person - was it possible to automatically spin up the react storybook along with the preact storybook in the `amp storybook` command?

Reply:
I was not able to get that working. For right now you need to use `amp storybook-react` to start it up